.remarkrc

```diff
@@ -5,6 +5,7 @@
     ["lint-list-item-indent", "tab-size"],
     ["lint-emphasis-marker", "consistent"],
     ["lint-maximum-line-length", false],
-    ["lint-maximum-heading-length", false]
+    ["lint-maximum-heading-length", false],
+    ["no-shortcut-reference-link", false]
```

Comment:
This doesn't seem to work the way I thought it would. The `remark` test still fails due to this specific warning.

Reply:
did you try adding:

```suggestion
    ["lint-no-shortcut-reference-link", false]
```
?

Reply:
That worked! Thanks!